ipaserver/install/dsinstance.py

```diff
@@ -997,16 +997,6 @@ def __add_delegation_layout(self):
     def __add_replication_acis(self):
         self._ldap_mod("replica-acis.ldif", self.sub_dict)
 
-    def __replica_ignore_initial_time_skew(self):
-        self.replica_manage_time_skew(prevent=False)
-
-    def replica_manage_time_skew(self, prevent=True):
-        if prevent:
-            self.sub_dict['SKEWVALUE'] = 'off'
-        else:
-            self.sub_dict['SKEWVALUE'] = 'on'
-        self._ldap_mod("replica-prevent-time-skew.ldif", self.sub_dict)
```

Comment:
You can remove this file, replica-prevent-time-skew.ldif. it's here and in the top-level Makefile.am.